tests/acceptance/test_project_servicehooks.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+
+from sentry.testutils import AcceptanceTestCase
+
+
+class ProjectTagsServiceHooksTest(AcceptanceTestCase):
+    def setUp(self):
+        super(ProjectTagsServiceHooksTest, self).setUp()
+        self.user = self.create_user('foo@example.com')
+        self.org = self.create_organization(
+            name='Rowdy Tiger',
+            owner=None,
+        )
+        self.team = self.create_team(organization=self.org, name='Mariachi Band')
+        self.project = self.create_project(
+            organization=self.org,
+            teams=[self.team],
+            name='Bengal',
+        )
+        self.create_member(
+            user=self.user,
+            organization=self.org,
+            role='owner',
+            teams=[self.team],
+        )
+
+        self.login_as(self.user)
+        self.path1 = '/settings/{}/{}/hooks/'.format(self.org.slug, self.project.slug)
+        self.path2 = '/settings/{}/{}/hooks/new/'.format(self.org.slug, self.project.slug)
```

Comment:
It'd be easier to read what's going on below if these were `self.hooks_path` and `self.new_hook_path`